Samples/Insights/cpp-win32/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Windows.ImplementationLibrary" version="1.0.210204.1" targetFramework="native" />
+  <package id="Microsoft.WindowsAppSDK.Foundation.TransportPackage" version="0.8.0-main.20210729.4.nightly" targetFramework="native" />

```

Comment:
>Microsoft.WindowsAppSDK.Foundation.TransportPackage [](http://example.com/codeflow?start=15&length=51)

this is internal. cannot use on public sample.

Reply:
Right. Until there is an official build to reference, this will have to do.